graphs/util/db.py

```diff
@@ -1608,6 +1608,300 @@ def find_all_graphs_containing_nodes(uid, search_type, search_word, view_type, d
 
 	return graph_dict.values()
 
+
+def uploadGPMLFile(username, graphJSON, title):
```

Comment:
This method is too long. Can you split it into more methods ? Going ahead we should have longer methods only if necessary. Try to keep method size to be atmost 15 statements. Atomic methods help improve the readability of the code.